src/pages/ContributionsPage/ContributionsPage.tsx

```diff
@@ -438,69 +442,73 @@ const ContributionsPage = () => {
           {!editHelpText ? (
             <Flex
               css={{
-                gap: '$md',
-                alignItems: 'center',
+                gap: '$sm',
+                alignItems: 'flex-end',
                 '@sm': { flexDirection: 'column', alignItems: 'start' },
               }}
             >
-              <Text p as="p">
-                {updatedContHelpText ? (
+              <MarkdownPreview
+                source={
                   updatedContHelpText
```

Comment:
instead of `a ? a : b ? b ? c` you can do `a ?? b ?? c`, here & for `updatedAllocText` below. maybe pull it out into a `const` defined in the component body for the sake of readability